unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/AssetBundles/AB/AssetPromiseKeeper_AB.cs

```diff
@@ -2,6 +2,9 @@ namespace DCL
 {
     public class AssetPromiseKeeper_AB : AssetPromiseKeeper<Asset_AB, AssetLibrary_AB, AssetPromise_AB>
     {
+        private static AssetPromiseKeeper_AB instance;
+        public static AssetPromiseKeeper_AB i { get { return instance ??= new AssetPromiseKeeper_AB(); } }
```

Comment:
I think this is the weirdest use of the `??` operator I ever seen.

Reply:
weird but beautiful at the same time 😅